### DISCUSSION TITLE
Compatibility with Creality Ender 3 Pro v1.1.4

### DISCUSSION THREAD

**Discussion by gschanuel** [luc-github/ESP3D#757] [Questions]
Hello there!
The Creeality v1.1.4 board is different from the one on the Wiki for [Creality CR 10 Ender 3](https://github.com/luc-github/ESP3D/wiki/Creality-CR-10-Ender-3) so I just want to be extra sure, Is it compatible?

**Also**, Is there any difference between soldering the ESP32 on the CH340G pins or on the ATMega1284p?

Thanks in advance!

![Creality v1.1.4](https://user-images.githubusercontent.com/2281677/168646196-b96299d4-5d54-49bb-9d5f-505e39bad17a.jpg)

**Comment by gschanuel** [comment]
OK.. it was easy :-)
The hard part was to flash the firmware into the ESP-Wroom-32
Then I found which version of ESP32 Arduino Core I should use in the README,md and everything was fine

**Comment by liqijian101** [comment]
【3D打印机 ESP3D无线模块 可以匹配任何打印机主板 无需修改固件 脱机打印-哔哩哔哩】 https://b23.tv/PfhS8ES
I test this

**Comment by luc-github** [comment]
@liqijian101 is your new version of SD Sharing board ?

**Reply by liqijian101** [reply]
yes ,this version can support the board that use sdio

**Reply by luc-github** [reply]
Nice do you sell some ? I do not have such solution to see how to improve SDIO - can you DM me on Discord ?

**Reply by liqijian101** [reply]
We have sales. Let me explain my plan . The wireless module uses the SPI to drive the TF card.I add a triode to control the TF power supply. When the module uploads the file, the TF card is powered down for 0.5 seconds and then powered back up. This allows the printer motherboard to drive the TF card using the sdio protocol.

**Reply by liqijian101** [reply]
I can send you a set if you provide a forwarding address. Another disadvantage of this scheme is that if the motherboard does not have a serial interface,only have usb serial, it can not be directly connected to the USB serial port and requires a fly-wire to the CH 340 pin. I have always been able to make a direct connection to the USB serial module. I had already made a pic solution, but the PIC chip was too expensive, so I abandoned the PIC solution.

**Reply by liqijian101** [reply]
DM? 
I’m sorry, I don’t know much about acronyms

**Reply by luc-github** [reply]
Direct Message ^_^

**Reply by liqijian101** [reply]
![Screenshot_20220518_090537](https://user-images.githubusercontent.com/16173343/168936400-68fc9ed4-0eb9-4959-96f9-ecb4097ecd39.jpg)

**Reply by luc-github** [reply]
@liqijian101 are you on https://discord.gg/e6zJ4aEU ?
